src/parser/types.ts

```diff
@@ -40,7 +57,24 @@ export interface StudyMetadata {
 /**
  * The UIConfig is used to configure the UI of the app.
  * This includes the logo, contact email, and whether to show a progress bar.
- * The UIConfig is also used to configure the sidebar, which can be used to display the task instructions and capture responses.
+ * The UIConfig is also used to configure the sidebar, which can be used to display the task instructions and capture responses. Below is an example of how the UI Config would look in your study configuration
+``` JSON
+  uiConfig:{
+    "contactEmail": "test@test.com",
+    "helpTextPath": "path/to/assets/help.md",
+    "logoPath": "path/to/assets/logo.jpg",
+    "withProgressBar": true,
+    "autoDownloadStudy": true
+    "autoDownloadTime": 5000,
+    "studyEndMsg": "Thank you for completing this study. You're the best!",
+    "sidebar": true,
+    "windowEventDebounceTime": 500,
+    "urlParticipantIdParam": "PROLIFIC_ID",
+    "numSequences": 500
+  }
+```
+In the above, the `path/to/assets/` path is referring to the path to your individual study assets. It is common practice to have your study directory contain an `assets` directory where all components and images relevant to your study reside. Note that this path is relative to the `public` folder of the repository - as is all other paths you define in reVISit.
```

Comment:
```suggestion
In the above, the `path/to/assets/` path is referring to the path to your individual study assets. It is common practice to have your study directory contain an `assets` directory where all components and images relevant to your study reside. Note that this path is relative to the `public` folder of the repository - as is all other paths you define in reVISit (unless you're using react components, in which case it's src/public).
```

Reply:
Edited in most recent commit. Just slightly different wording.